src/app/core/services/api/fake-backend/connector-fake-impl/data/test-assets.ts

```diff
@@ -73,6 +73,21 @@ export namespace TestAssets {
 This is a short description text that should be fully rendered without being **collapsed**. No *show more* button should be visible.
 `;
 
+  export const assetWithCustomProperties: UiAsset = {
+    dataSourceAvailability: 'LIVE',
+    assetId: 'asset-with-custom-properties',
+    title: 'Asset with Custom Properties',
+    description: 'Asset with Custom Properties',
+    descriptionShortText: 'Asset with Custom Properties',
+    connectorEndpoint: 'https://my-connector/api/dsp',
+    participantId: 'MDSL1234XX.C1234XX',
+    creatorOrganizationName: 'My Org',
+    isOwnConnector: true,
+    customJsonLdAsString: '{"http://custom.json.test/LD":"https://google.com"}',
+    customJsonAsString:
```

Comment:
Custom Json (without LD) as string should have a property where the key is not an url, e.g. "customProp": "customValue"